src/main/java/org/jenkinsci/test/acceptance/po/FormValidation.java

```diff
@@ -67,23 +67,24 @@ public static FormValidation await(Control control) {
 
         // Special handling for validation buttons and their markup
         if (element.getTagName().equals("button")) {
-            WebElement spinner = element.findElement(control.by.xpath("./../../../following-sibling::div[1]"));
+            WebElement spinner = element.findElement(by.xpath("./../../../following-sibling::div[1]"));
```

Comment:
was `control` affecting by any means?

Reply:
https://github.com/jenkinsci/acceptance-test-harness/pull/646/files#diff-6272a530cea8f96e9ff459b762ff966c5a25650cfaf3229431b06bec54dfd269R37

by is a static field of `CapybaraPortingLayer` of which `Control` extends,  So this is unrelated to the fix - just removing the `"Field by of blah should be accessed in a static way"` linting.